go/app/main.go

```diff
@@ -51,6 +126,61 @@ func getImg(c echo.Context) error {
 	return c.File(imgPath)
 }
 
+func updateJson(name string, category string, image *multipart.FileHeader) {
+	hashedFileName := sha256.Sum256([]byte(image.Filename))
+
+	jsonFile, err := os.Open(ItemFile)
+	if err != nil {
+		fmt.Println("Cannot open the json file: ", err)
+		return
+	}
+	defer jsonFile.Close()
+
+	jsonData, err := ioutil.ReadAll(jsonFile)
+	if err != nil {
+		fmt.Println("Cannot read data: ", err)
+		return
+	}
+
+	var items Items
+
+	json.Unmarshal(jsonData, &items)
+	items.Items = append(items.Items, Item{Name: name, Category: category, ImageFileName: fmt.Sprintf("%x.jpg", hashedFileName)})
```

Comment:
Maybe it'll be better to obtain the file extension `.jpg` directly from the file!